README.md

```diff
@@ -41,18 +41,199 @@ action:
 
 ```
 
-***NOTE:*** For database action, remember to set up your Aito instance credentials.
+***NOTE:*** For the database action, remember to set up your Aito instance credentials.
 
-For addition guide of the CLI tool, see the [CLI documentations](docs/cli.md)
+For an additional guide of the CLI tool, see the [CLI documentations](docs/cli.md)
 
-### Integrating with [pandas](https://pandas.pydata.org/) DataFrame
+### Getting your data into Aito using Python
+
+To get you data into Aito you will have to go through the following steps:
+
+1. [Define Aito schema](#1-define-aito-schema)
+2. [Upload Aito schema](#2-upload-aito-schema)
+3. [Upload data](#3-upload-data)
+
+#### 1. Define Aito schema 
+
+You can either create the schema manually, by using your own code or use the infer feature of the Aito python library.
+
+You can use [pandas](https://pandas.pydata.org/) DataFrame to infer your Aito schema.
+
+For more information on the Aito schema can be found [here](https://aito.ai/docs/articles/defining-a-database-schema/)
+
+##### Infer Aito Schema from a pandas DataFrame
+
+
+You can download the example file from [here](https://raw.githubusercontent.com/AitoDotAI/kickstart/master/reddit_sample.csv).
 
-* Infer Aito Schema from a pandas DataFrame:
   ```python
+  import pandas as pd
   from aito.utils.schema_handler import SchemaHandler
 
+  # Load a CSV into a pandas Dataframe
+  dataframe = pd.read_csv("reddit_sample.csv", delimiter=";")
```

Comment:
```suggestion
  dataframe = pd.read_csv("reddit_sample.csv")
```

Reply:
In the notebook I created for Pastest I had to define the "delimiter" since the csv was ported from excel with ";" as the delimiter. If the csv is correctly formated, this would mess it up. So it's better to leave it out.